spec/support/spec_file_name_enforcer.rb

```
@@ -0,0 +1,15 @@
+RSpec.configure do |config|
+  config.before(:suite) do
+    
+    files_without_spec_suffix = Dir.glob('spec/**/*').select do |filename|
+      File.file?(filename) && 
+        !filename.match(/_spec\.rb\z/) &&
+        !filename.match(%r{\Aspec/(support|factories)/}) &&
+        !filename.match(%r{\Aspec/(spec_helper\.rb|rails_helper\.rb|examples\.txt)\z})
+    end
+    
+    if files_without_spec_suffix.any?
+      raise 'Spec files need _spec.rb suffix: ' + files_without_spec_suffix.join(', ')
+    end
+  end
+end
```

@mhgoldman Spec files without _spec.rb suffix now cause the specs to fail early